include/clang/Basic/DiagnosticIDs.h

```diff
@@ -34,7 +34,7 @@ namespace clang {
       DIAG_SIZE_SERIALIZATION =  120,
       DIAG_SIZE_LEX           =  400,
       DIAG_SIZE_PARSE         =  500,
-      DIAG_SIZE_AST           =  110,
+      DIAG_SIZE_AST           =  120,
```

Comment:
This change is required after the last merged pull request to make clang compile.